packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -142,11 +142,17 @@ export async function fetchHelper(
         // It could container PII, like URI in message itself, or token in properties.
         // It is also non-serializable object due to circular references.
         //
-        const failureCode = online === OnlineStatus.Offline ? offlineFetchFailureStatusCode : fetchFailureStatusCode;
-        throwOdspNetworkError(
-            `odspFetchThrewError [${failureCode}]`,
-            failureCode,
-        );
+        if (online === OnlineStatus.Offline) {
+            throwOdspNetworkError(
+                `Offline`,
+                offlineFetchFailureStatusCode,
+            );
+        } else {
+            throwOdspNetworkError(
+                `Fetch error`,
```

Comment:
I find these generic error phrases so useless in telemetry.  Can we stick with something more explicit?

Related - we are not on the same page about fluidErrorCode at this point.  I haven't had time to give it continued thought but will try to prioritize it so we are moving forward harmoniously.

Until then, can you stick with the `camelCase` format for this parameter?

Reply:
Ok I see your change to `throwOdspNetworkError` below.  Can we exclude this and that change for now and leave it to #8258?  I want to avoid extra churn in the strings going to our telemetry DBs because as you know it makes extra cases to account for in queries (or mentally when looking at results)

Reply:
I've undid throwOdspNetworkError() changes, and change usage here, but kept the same idea